### ISSUE TITLE
[Bug] - POST call to sign in to a page doesnt work on postman windows app but works in postman chrome app

### ISSUE THREAD

**Issue by tariqghalib** [a85/POSTMan-Chrome-Extension#2577]
**Bug Description:**
If I do a post call to login by basically providing username and password it works under the postman chrome app but it fails with postman windows app.

**Version/App Information:**
-->
 1. Postman Version: 4.9.2
 2. App (Chrome app or Mac app): Windows
 3. OS details: Windows 64 professional
 4. Is the Interceptor on and enabled in the app: Yes
 5. Did you encounter this recently, or has this bug always been there: Recently
 6. Expected behaviour: We should be able to login through a post call providing login credentials through postman windows app as well

1. For example run a command
curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -H "Postman-Token: 81e3c9c6-f718-e32d-66f0-bc3" "https://login.<xyz>.com/<authentication>/<xyz>.xhtml?id=<ID>&response_type=<Static>&redirect_uri=<redirect_uri>&Name=<ContextName>&un=<testuser>&pw=<password>"

This throws an error in the Postman Windows app:
<div id=<Error>>
          Your login credentials could not be resolved.  Please try logging in again.
</div>

While this works fine in the Chrome app.

**Comment by vegetableman** [comment]
@tariqghalib Could you verify and compare the `POST` request being sent through `Postman Console` on the native app and devtools on the chrome app? 
Checkout https://www.getpostman.com/docs/errors for more details.

**Comment by tariqghalib** [comment]
@vegetableman I have tried it but on postman chrome I dont get any console data . I have even tried to get some Network data but nothing useful there. On chrome I do get success though but on Windows app, it just fails with error that I mentioned above.
One observation is if I turn OFF interceptor on chrome it doesnt gives me any response back from the login page. Is there anything related to that missing on postman windows app?

**Comment by tariqghalib** [comment]
@vegetableman Actually I investigated a bit further by building the whole request through nodejs and playing around with query string parameters and I found that:

1. Our sign in (web app) requires few 302 redirects in the process of authentication 
2. Chrome was able to preserve all the query string parameter values during the 302 redirect but windows app actually does not preserve (or changes) the parameters and final referrer location during the redirects. This causes the redirect to fail and results in a failed api call overall.
